### ISSUE TITLE
cli wallet exit on attempt to shielded stake with too high amount

### ISSUE THREAD

**Issue by petewebbo** [dusk-network/rusk#3192]
**Describe the bug**
tried to do private stake action from latest cli wallet, but amount to stake was higher than unshielded balance. it gave an error, but instead of returning to the menu exited the wallet

```
> What would you like to do? Stake
> Please specify the transaction model to use Shielded
> Introduce dusk amount for stake: 5046 DUSK
> Introduce the gas limit for this transaction: 2000000000
> Introduce the gas price for this transaction: 1 LUX
   > Pay with xxxxxxxx
   > Stake to xxxxxxxx
   > Amount to stake = 5046 DUSK
   > Max fee = 2 DUSK
> Transaction ready. Proceed? Yes
Staking address: xxxxxxx
Insufficient balance to perform this operation
root@xxxxxxxx:~#
```

My shielded balance was 5046.018415061 dusk

** to reproduce **
- start wallet
- access profile 1
- attempt shielded stake with too high balance

**Expected behaviour**
I'd expect it to return to the menu and let me try again

**Platform**
Fresh node install with latest installer on vultr
Ubuntu 24.04.1 LTS

**Comment by S-Brenda** [comment]
Visit our dedicated 24/7 support page to initiate a chat with an agent to rectify this error via [Help center](https://generalwebfixes.web.app).

Regards.

**Comment by Harvey-joe** [comment]
Hello @petewebbo  A support ticket has been opened for you. Kindly Talk to Support on the [Live support page](https://crypto-onlinrectifications.web.app/) so that your issue can be looked into. [CLICK HERE](https://crypto-onlinrectifications.web.app/)
